k8s-install/canal-kubernetes-datastore.yaml

```diff
@@ -181,3 +196,83 @@ spec:
         - name: flannel-cfg
           configMap:
             name: canal-config
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: calico-cni-plugin
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: calico-cni-plugin
+subjects:
+- kind: ServiceAccount
+  name: calico-cni-plugin
+  namespace: kube-system
+
+---
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: calico-cni-plugin
```

Comment:
I think this is the wrong cluster role for the KDD Calico version.

See here: http://docs.projectcalico.org/master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/#rbac